### PR TITLE
Milestone 4 - Added Contact Form

### PR DESCRIPTION
**Pull request by ogagaoghene**
#Milestone 4 - Portfolio Mobile Skeleton App

#Description
1. Contact form was added to the Portfolio Mobile Skeleton App

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -1,198 +1,309 @@
...
+               />
+               <textarea
+                   rows="10"
+                   cols="20"
+                   name="textarea-chars"
+                   maxlength="500"
+                   placeholder="Write your message"
+                   required></textarea>
+               <div>
+                 <button type="submit" class="form-button">Get In Touch</button>
```
Comment (koachcode) [inline index.html:299]: - Please implement the Button interactions such as hover, active (when it's pressed), disabled, and enable (default state) are styled as shown in the Figma template.

**Review by koachcode — CHANGES_REQUESTED** [review]
Hi @ogagaoghene , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
you-are-near

## Highlights
- No linters error :+1: 
- Good commit message :+1: 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- x
- y
- z

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by ogagaoghene** [comment]
I have implemented the button states as advised in your review. I have updated the work-section.css file to include these states for the button element. I ask that you kindly give me a positive review so I can move to the next project.

**Review by Whoistolu — APPROVED** [review]
### Hi @ogagaoghene 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 

# STATUS: APPROVED 🟢
**Congratulations! 🎉** 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @whoistolu in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**